engine.py

```diff
@@ -0,0 +1,35 @@
+import models
+import setting
+
+
+def get_player_name() -> str:
+    name = ''
+    while not name:
+        name = str(input('Please input your name ')).strip()
```

Comment:
`input` завжди повертає `str`
```suggestion
        name = input('Please input your name ').strip()
```